core/src/com/unciv/logic/battle/BattleDamage.kt

```diff
@@ -166,11 +173,15 @@ object BattleDamage {
             )
                 modifiers["Statue of Zeus"] = 15
         } else if (attacker is CityCombatant) {
-            if (attacker.getCivInfo()
-                    .hasUnique("+50% attacking strength for cities with garrisoned units")
-                && attacker.city.getCenterTile().militaryUnit != null
-            )
-                modifiers["Oligarchy"] = 50
+            if (attacker.city.getCenterTile().militaryUnit != null) {
+                var OlichargyBonus = 0;
```

Comment:
AS will complain and say: var names please start with lower case - AS has a 'Problems' view and I tend to keep it empty if possible - makes the tool more useful when it's suddenly no longer empty.

Reply:
In this case I agree with the tool, since this doesn't affect readability it's best to stick with the official kotlin style